doc/tutorials/acrn_on_qemu.rst

```diff
@@ -186,10 +188,13 @@ Install ACRN Hypervisor
          insmod ext2
 
          echo 'Loading ACRN hypervisor ...'
-         multiboot --quirk-modules-after-kernel /boot/acrn.32.out
+         multiboot --quirk-modules-after-kernel /boot/acrn.32.out  root=/dev/vda5
          module /boot/bzImage Linux_bzImage
       }
 
+     .. note::
+     If your rootfs partition isn't vda5, please change it to match with yours.
+
 #. Update GRUB:
 
    .. code-block:: none
```

Comment:
At the end of the tutorial, add a step that confirms that the User VM L2 guest is running successfully.

Reply:
> At the end of the tutorial, add a step that confirms that the User VM L2 guest is running successfully.

@amyreye  User VM L2 guest **login** will show up if it run successfully.